combat-trainer.lic

```diff
@@ -654,8 +699,9 @@ class SpellProcess
     @before.each { |action| bput(action['message'], action['matches']) } if @before
 
     @custom_cast = "cast my #{game_state.weapon_name}" if game_state.casting_ignite
+    @custom_cast = "cast #{game_state.healing_target}" if game_state.casting_consume
     case bput(@custom_cast || 'cast', @casts)
-    when /^Your target pattern dissipates/, /^You can't cast that at yourself/
+    when /^Your target pattern dissipates/, /^You can't cast that at yourself/, /^You need to specify a body part to consume./
```

Comment:
The other two are already in base.yaml.

I ask about the new message because it appears to be checking against the result of `bput`, and I wasn't sure how bput would return that string without it being in `@casts` (which comes from base.yaml).